daphne/src/vdaf/mod.rs

```diff
@@ -385,7 +388,8 @@ impl VdafConfig {
                 extensions: vec![],
                 payload: encoded_input_share,
             },
-            _ => PlaintextInputShare::get_decoded(&encoded_input_share)?,
+            _ => PlaintextInputShare::get_decoded(&encoded_input_share)
+                .map_err(|e| fatal_error!(err = e))?,
```

Comment:
`DapError::Fatal` is intended for situations where the host encountered an unrecoverable error of its own making. In this particular situation, the error is triggered by the peer, so the appropriate action is to abort. I.e., I believe the current behavior is to map the CodeError into a `DapError::Abort`.

Reply:
I'll review, but if I made this change it's because the compiler forced me to, which came about from me deleting the from conversions for DapError. Which means this was already wrong to begin with. I'll change it to an abort

Reply:
On `main` we have this:
```rust
impl From<CodecError> for DapError {
    fn from(e: CodecError) -> Self {
        Self::Fatal(format!("codec: {e}"))
    }
}
```
I believe you just need to add that back?

Reply:
Ah, which is incorrect: It should be `Self::Abort`.